microsetta_public_api/tests/test_resources_alt.py

```diff
@@ -0,0 +1,218 @@
+from unittest import TestCase
+from unittest.mock import patch, Mock
+
+from microsetta_public_api.resources_alt import Component
+
+
+class ComponentTests(TestCase):
+
+    def test_init(self):
+        comp = Component('some_name')
+        self.assertEqual(comp.name, 'some_name')
+        self.assertDictEqual(comp.children, dict())
+        self.assertIsNone(comp.data)
+
+    def test_add_child(self):
+        comp = Component('some_name')
+        child1 = Component('child1')
+        child2 = Component('child2')
+        child3 = Component('child3')
+        comp.add_child(child1)
+        child1.add_child(child2)
+        child1.add_child(child3)
```

Comment:
I don't think parent pointers are set child1, child2, child3?  If this is an invalid state, we probably shouldn't allow it.  If we don't actually care about parent pointers, we should delete them.

Reply:
Fixed in https://github.com/biocore/microsetta-public-api/pull/57/commits/bf278b80b5b898af365f5590f1bf2d8ca7254b32